pull_request_template.md

```diff
@@ -2,7 +2,7 @@
 - *Description of why you made this PR*
 
 ## Solution
-- *And how you fix that*
+- *And how do you fix that problem*
```

Comment:
```suggestion
- *And how you fix that problem*
```
I'd still prefer the affirmative form: "Description of why you made this PR and how you fix that problem"

Reply:
That's the template actually, https://github.com/YunoHost/apps/blob/master/pull_request_template-HQ-apps.md
Which doesn't mean we can't change it here, but would be better to fix the template if we need to so it will spread.